includes/admin/tools/export/class-batch-export-donors.php

```diff
@@ -223,7 +223,8 @@ public function get_data() {
 				);
 			}
 
-			$payments = ( new Give_Payments_Query( $args ) )->get_payments();
+			$payments_query = new Give_Payments_Query( $args );
```

Comment:
@ravinderk Sure. I've already put this in my checklist to test each issue after fixing using lower versions of php. Also, i'm fixing the issue using lower php versions now.